internal/build/docker_test.go

```diff
@@ -19,14 +18,14 @@ import (
 	"time"
 
 	"github.com/windmilleng/tilt/internal/model"
+	"github.com/windmilleng/tilt/internal/test_utils"
```

Comment:
package names should not have underscores
https://golang.org/doc/effective_go.html#package-names